os/board/rtl8730e/src/component/os/tizenrt/rtk_netmgr.c

```diff
@@ -814,6 +910,15 @@ trwifi_result_e wifi_netmgr_utils_set_chplan(struct netdev *dev, uint8_t chplan)
 	return wuret;
 }
 
+trwifi_result_e wifi_netmgr_utils_get_disconn_reason(struct netdev *dev, int *deauth_reason)
+{
+	trwifi_result_e wuret = TRWIFI_FAIL;
+	*deauth_reason = wifi_get_last_reason();
+	printf("wifi_netmgr_utils_get_disconn_reason : %d\n", *deauth_reason);
```

Comment:
This change was from Samsung, is this printf line needed?

Reply:
No, This is line was for debugging. Not needed. Can you remove this?